### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ArturVyklynets**
- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] **color-text** is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,69 @@
...
+          </li>
+
+          <li class="nav-item">
+            <a
+             class="nav-link"
+             href="#"
+             data-qa="hover"
+            >
+            Laptops & Computers
+            </a>
```
Comment (VitaliyBodnar) [inline src/index.html:45]: code it not aligned
```suggestion
            <a
              class="nav-link"
              href="#"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+  display: flex;
+  list-style: none;
+}
+
+.nav-item{
+  margin-right: 20px;
+}
+
+.nav-item:nth-last-child(1){
+margin-right: 0;
```
Comment (VitaliyBodnar) [inline src/style.css:40]:
```suggestion
  margin-right: 0;
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,69 @@
...
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav-item{
+  margin-right: 20px;
+}
+
+.nav-item:nth-last-child(1){
```
Comment (VitaliyBodnar) [inline src/style.css:39]:
```suggestion
.nav-item:last-child {
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,69 @@
...
+margin-right: 0;
+}
+
+.nav-link{
+  height: 60px;
+  color:#000;
+  text-decoration: none;
+  display: flex;
+  align-items:center;
+}
```
Comment (VitaliyBodnar) [inline src/style.css:49]: missed spaces
```suggestion
.nav-link {
  height: 60px;
  color: #000;
  text-decoration: none;
  display: flex;
  align-items: center;
}
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,69 @@
...
+  margin-right: 20px;
+}
+
+.nav-item:nth-last-child(1){
+margin-right: 0;
+}
+
+.nav-link{
+  height: 60px;
+  color:#000;
```
Comment (VitaliyBodnar) [inline src/style.css:45]: It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

**Review by loralevitska — APPROVED** [review]
Well done)

**Pull request closed without merging**